permission_handler_platform_interface/lib/src/permissions.dart

```diff
@@ -94,13 +94,27 @@ class Permission {
   /// Depending on the platform and version, the requirements are slightly
   /// different:
   ///
-  /// **Android:**
-  /// - When running on Android 13 (API 33) and above: Read image files from
-  /// external storage
-  /// - When running below Android 13 (API 33): Nothing
-  ///
   /// **iOS:**
   /// - When running Photos (iOS 14+ read & write access level)
+  ///
+  /// **Android:**
+  /// - Devices running Android 13 (API level 33) and above: use [Permissions.storage].
+  /// - Devices running Android 12 (API level 32) or lower: Should use [Permissions.photos].
```

Comment:
If I remember correctly the `storage` permissions have been updated starting from Android 11 and instead of requesting `storage` permissions you now have to request `photos`, `video` or `audio` depending on the applications need.

Requesting `storage` permissions will request `READ_EXTERNAL_STORAGE` permission which is no longer allowed on Android 11 and higher (documentation: https://developer.android.com/about/versions/11/privacy/storage).